### PR TITLE
refactor(alerts): fix typo and styles of `alerts` examples

### LINKED ISSUE #219: Fix `Alerts` component examples
### Subject

Alerts

### Description

There is an extra sentence in the `Alerts` component examples in `with-list`.
```diff
        <AlertTitle mb={3}>
-          There were 2 errors with your submission Your password must be at least.
+          There were 2 errors with your submission.
        </AlertTitle>
```
There is also "FIXME" and should have `maxW` as they are too wide.
The arrow in the details link could be improved because it is positioned lower than centre.

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #219

## Description

This fixes the typo and styles of `Alerts` component examples.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

**Review by quantumshiro — APPROVED** [review]

File: contents/feedback/alerts/with-link-on-right/index.tsx [hunk 1]
```diff
@@ -3,14 +3,13 @@ import type { FC } from "react"
 
 const WithLinkOnRight: FC = () => {
   return (
-    <Alert status="info" variant="subtle">
-      {/* FIXME: Iconをwarningに設定 */}
+    <Alert status="info" variant="subtle" maxW="5xl">
```
Comment (Tomoya-Matsubara) [inline contents/feedback/alerts/with-link-on-right/index.tsx:6]: Shouldn't `status` be `warning`, considering `FIXME`?

```diff
- <Alert status="info" variant="subtle" maxW="5xl">
+ <Alert status="warning" variant="subtle" maxW="5xl">
```

Comment (taroj1205) [reply contents/feedback/alerts/with-link-on-right/index.tsx:6]: Actually, I forgot to remove the comment, it should be `info`.

Comment (Tomoya-Matsubara) [reply contents/feedback/alerts/with-link-on-right/index.tsx:6]: I see. Probably the `FIXME` comment was wrong, right?

Comment (taroj1205) [reply contents/feedback/alerts/with-link-on-right/index.tsx:6]: I think so.

Comment (Tomoya-Matsubara) [reply contents/feedback/alerts/with-link-on-right/index.tsx:6]: I got it. Conversation resolved :)

**Review by Tomoya-Matsubara — APPROVED** [review]

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**